src/components/section.js

```diff
@@ -1,15 +1,14 @@
 export default class Section {
```

Comment:
`section.js` should be in `UpperCase` like the other files with classes, according to the checklist